gtsfm/evaluation/metrics.py

```diff
@@ -0,0 +1,287 @@
+"""Class to store metrics computed in different GTSfM modules.
+
+Authors: Akshay Krishnan
+"""
+from __future__ import annotations
+
+import json
+import numpy as np
+from enum import Enum
+from typing import Any, Dict, List, Union
+
+import gtsfm.utils.io as io
+
+DATA_KEY = "full_data"
```

Comment:
Could you provide some high level comments explaining what these keys mean? How is full data defined, and when is it preferred?

Reply:
done